src/applications/hca/tests/config/general.unit.spec.jsx

```diff
@@ -10,7 +10,7 @@ import {
 } from 'platform/testing/unit/schemaform-utils.jsx';
 import formConfig from '../../config/form';
 
-describe('Hca general insurance', () => {
+describe.skip('Hca general insurance', () => {
```

Comment:
We should not be skipping test, only for debugging, need to fix test before merging so our coverage is maintained.

Reply:
I have resolved this issue with a caveat that I'm still learning React Testing a bit. I figured out that general.unit needed to be broken up into general.unit AND providerAndPolicy.unit.

Reply:
All good no worries.